README-sycl.md

```diff
@@ -229,12 +229,11 @@ source /opt/intel/oneapi/setvars.sh
 # Build LLAMA with MKL BLAS acceleration for intel GPU
 mkdir -p build && cd build
 
-# Option 1: Use FP16 for better performance in long-prompt  inference
-cmake --build .. -DLLAMA_SYCL=ON -DCMAKE_C_COMPILER=icx -DCMAKE_CXX_COMPILER=icpx -DLLAMA_SYCL_F16=ON
-# Or without "--build", run "make" next
+# Option 1: Use FP16 for better performance in long-prompt inference
+#cmake .. -DLLAMA_SYCL=ON -DCMAKE_C_COMPILER=icx -DCMAKE_CXX_COMPILER=icpx -DLLAMA_SYCL_F16=ON
```

Comment:
This line should not start with `#`.

Reply:
I see, excellent observation. That is not a good way to notate that, though. I think the `#` should be removed and add to the comment above something like: `. Not generally recommended`.

Reply:
In fact, this cmd is copied from the example/sycl/build.sh at beginning.
But next editor don't know it and modify it again and again. :)